Add PM2 tutorial series to resources

marcuspoehls opened the pull request:
Add the [PM2 tutorial series](https://futurestud.io/tutorials/pm2-utility-overview-installation) by Future Studio as a resource within the Readme

vmarchaud commented:
Good series but there are some inaccurate tutorials, for example `pm2 startup` that we refactored in `pm2 v2.2` and the graceful start/stop that has been rewrited to be simpler.

For the startup, you just need to replace platform by init-system : `systemd`, `systemv`, `upstart`, `launchd` and update the corresponding output.
Graceful start/stop is documented right [there](http://pm2.keymetrics.io/docs/usage/signals-clean-restart/)

Feel free to contact me over our slack (http://slack.pm2.io) if you want some help.

marcuspoehls commented:
@vmarchaud TL;DR: all tutorials are up-to-date.

The initial version of each guide has been written using PM2 `0.14.3`. Now with `2.2.2` available, I’ve updated each guide to the changes introduced within the latest major version. 

While reviewing all tutorials, I’ve added multiple ideas to my “to-write-list”. Means, new posts will be added in the upcoming months.

Thank you for your valuable input to improve the available tutorials!

vmarchaud commented:
Could you remake this PR against master ? Our dev branch is full of R&D work and will not be merged for a while.
